css/style.css

```diff
@@ -431,4 +431,29 @@ video {
   .footer__icon {
     margin: 19px;
   }
+
+  .info {
+    margin-bottom: 18px;
+    width: 100%;
+    left: 0;
+    right: 0;
+    margin: auto;
+    text-align: center;
+  }
+  
+  .info-title {
+    width: 227px;
+    height: 24px;
+    font-size: 20px;
+    left: 0;
+    right: 0;
+    margin: auto;
+  }
+  
+  .info-description {
```

Comment:
I think this class should be called info__description

Reply:
changed, thanks to opinion